Tests/LibWeb/Ref/input/wpt-import/css/css-text-decor/text-decoration-propagation-04.html

```
@@ -0,0 +1,24 @@
+    ipsum
+  </div>
+  dolor
+  <div style="text-decoration: overline">
+    sit
+    <div style="text-decoration: line-through">
+      amet
+    </div>
+    consectetur
+  </div>
+  adipiscing
+</span>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </span> ]

(tag-pair)